inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarOperator.java

```diff
@@ -152,7 +152,7 @@ public void createTopic(PulsarAdmin pulsarAdmin, PulsarTopicBean topicBean) thro
         }
 
         try {
-            if (PULSAR_QUEUE_TYPE_SERIAL.equals(topicBean.getQueueModule())) {
+            if (PULSAR_QUEUE_TYPE_SERIAL.equalsIgnoreCase(topicBean.getQueueModule())) {
```

Comment:
It is recommended to standardize the parameters passed externally instead of making different types of case compatibility on the server side.

Reply:
done